app/controllers/admin/admin_users_controller.rb

```diff
@@ -71,5 +72,15 @@ def authorize_user!
 
       raise ApplicationController::NotAuthorized
     end
+
+    def load_admin_email
+      begin
+        user =
+          Gravity.client.user(id: @admin_user.gravity_user_id).user_detail._get
```

Comment:
**question (non-blocking):** Is there a Metaphysics endpoint that we can use instead of directly requesting user information from Gravity? Ideally all requests between Artsy services go through Metaphysics, and I'd like to see if we can start following that pattern in Convection.

Reply:
We added a Metaphysics client at the [PR](https://github.com/artsy/convection/pull/1221) but it's not merged. For now, Convection doesn't use Metaphysics to load information as far as I know.